### PR TITLE
Minor content corrections

### PR DESCRIPTION
**Pull request by scr0llbaer9000**
Some minor things I found during playtesting.

Some `staff` `function` corrections.

Some adjustments to some `targetgroupattractivity` values.

`Neid-Reiter` for consistency (had long already been used in news items, I had forgotten about the German version; though I don't find that spoof title all too good).

Shortened some too long descriptions (of old items) to fit into the available space.

### CODE REVIEW

File: res/database/Default/user/jorgaeff.xml [hunk 1]
```diff
@@ -1004,12 +1004,12 @@
 
 		<news id="news-jorgaeff-dyson-05" type="2" thread_id="news-jorgaeff-dyson" creator="8936" created_by="jorgaeff">
 			<title>
-				<de>Dyson vs. Klatschkowski: Tumulte</de>
-				<en>Dyson vs. Klatschkowski: turmoil</en>
+				<de>Dyson vs. Klatschkowski: Tumulte vor Boxkampf</de>
+				<en>Dyson vs. Klatschkowski: Riots before boxing</en>
```
Comment (nittka) [inline res/database/Default/user/jorgaeff.xml:1008]: before match?

**Review by nittka — CHANGES_REQUESTED** [review]
Apart from the one question no objections. Let me know if you want to keep the PR open for potential furter fixes.

File: res/database/Default/user/jorgaeff.xml (hunk 1 above)
Comment (scr0llbaer9000) [reply res/database/Default/user/jorgaeff.xml]: fine by me.

**Comment by scr0llbaer9000** [comment]
Wasn't there a release planned soon?

We can keep this open for things I come across until then. No big additions.

Maybe country `MEX` has to be added now though.

**Review by nittka — APPROVED** [review]
I will wait still with merging in case you have more fixes before the release.

**Pull request merged**